### PR TITLE
Docs/pipeline docs update

### PR DESCRIPTION
**Pull request by subdavis**

### CODE REVIEW

File: docs/Pipeline-Documentation.md [hunk 1]
```diff
@@ -52,12 +60,36 @@ An assortment of other types of utility pipelines.  Utility piplines are named `
 
 ## Training
 
-Run model training on ground truth annotations you've created in order to generate a reusable pipe.
+Run model training on ground truth annotations.  Currently, training configurations are available to do object detection, object classification, and full-frame classification.  Tracker training will be added in a future update.
+
+### Important notes
+
+* Object detector training configurations require annotations labeled `fish`.
```
Comment (BryonLewis) [inline docs/Pipeline-Documentation.md:67]: If you train over generic detections I don't think you need the 'fish' label?

Comment (subdavis) [reply docs/Pipeline-Documentation.md:67]: Right, the default is to train on all labels encountered in groundtruth unless `--labels` are specified.

I forgot that `--input` is an older style and we have `-il, -it` now.  the `fish` I linked in slack was just an example placeholder.

The question remains though: Do _any_ of the training configurations require a "fish" label?  Which ones can take arbitrary ground truth, and which ones require specific labels?

File: docs/Pipeline-Documentation.md [hunk 2]
```diff
@@ -1,44 +1,52 @@
...
 | Pipeline | Use case |
 | -------- | -------- |
 | <pre>em tuna</pre>  | tracker |
 | <pre>fish</pre>  | simple fish tracker |
 | <pre>fish.sfd</pre>  | tracker |
-| <pre>generic</pre>  | tracker |
-| <pre>motion</pre>  | tracker |
-| <pre>mouss</pre>  | tracker |
-| <pre>sefsc bw *</pre>  | black-and-white multi-class fish |
+| <pre>generic</pre>  | generic object tracker for bootstrapping groundtruth annotations |
```
Comment (mattdawkins) [inline docs/Pipeline-Documentation.md:46]: all pipelines can bootstrap groundtruth annotations this isn't unique to generic detector and generic detector is not something you should encourage people to run over the others necessary, I would just say "Puts classless boxes around arbitrary objects" or the like

File: docs/Pipeline-Documentation.md (hunk 1 above)
Comment (mattdawkins) [reply docs/Pipeline-Documentation.md:67]: If you train over anything you shouldn't need the word fish no models would require it

File: docs/Pipeline-Documentation.md [hunk 3]
```diff
@@ -52,12 +60,36 @@ An assortment of other types of utility pipelines.  Utility piplines are named `
...
+* Object detector training configurations require annotations labeled `fish`.
+* Full-frame classifiers can be trained on arbitrary multi-class labels.  It's helpful to start with `empty frame lbls` utility pipe and add type annotations to each generated frame.
+* Object classifiers are trained on bounding boxes with arbitrary multi-class labels.
+
+### Overview
+
+* SVM ([Support Vector Machine](https://en.wikipedia.org/wiki/Support-vector_machine)) configurations are usable with the smallest amount of ground-truth and train relatively quickly.
+* [NetHarn](https://gitlab.kitware.com/computer-vision/netharn) is a pytorch deep learning framework that requires more input data: on the order of thousands of target examples.  There are two architectures used.  Netharn models can take up to several days to train.
+    * Cascade Faster R-CNN (cfrnn)
+    * ResNet (Residual Network)
```
Comment (mattdawkins) [inline docs/Pipeline-Documentation.md:76]: Also mask Rcnn might want to say mask rcnn useful for pixel class + detection (boxes), CFRNN just detection (boxes), resnet full frame or secondary object classification

**Comment by subdavis** [comment]
Comments addressed.  I've already deployed to https://kitware.github.io/dive/Pipeline-Documentation/ but if I need to make more tweaks i can.  I think this gets everything though.

**Review by BryonLewis — APPROVED** [review]

**Pull request merged**